src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -24,12 +24,12 @@
        <number>10</number>
       </property>
       <item row="5" column="0">
-       <widget class="QLabel" name="labelPlayingTrackProtection">
+       <widget class="QLabel" name="labelLoadWhenDeckPlaying">
         <property name="text">
-         <string>Playing track protection</string>
+         <string>Loading a track, when deck is playing</string>
         </property>
         <property name="buddy">
-         <cstring>checkBoxDisallowLoadToPlayingDeck</cstring>
+         <cstring>comboBoxLoadWhenDeckPlaying</cstring>
```

Comment:
I'm not sure why you changed this (changing translation texts requires new translation for every language).

Also, I thought the new option would a) load a track into a playing deck and b) play that right away.
Even though b) requires a), a) and b) are unrelated (I might want to load tracks into playing decks, but I don not want auto-play)

Reply:
It's actually deleting the checkbox entirely and replacing it with a combobox that offers more functionality:

1. load track into playing deck and play that right away
2. load track into playing deck but stop
3. don't allow loading into playing deck

See the discussion at the end of this issue? https://github.com/mixxxdj/mixxx/issues/10548#issuecomment-1268685304

Reply:
I should have taken a closer look, not just skimming through...